app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,290 @@
+     * @return integer
+     */
+    public function sendMail(Request $request, $email)
+    {
+        $mailSent = $this->mail->send('emails.adminInvite', ['username' => $request->username, 'token' => $request->_token], function ($message) use ($email) {
+                $message->from(getenv('SENDER_ADDRESS'), getenv('SENDER_NAME'));
+                $message->to($email->email)->subject('Suyabay Invitation');
+            });
+
+        if ($mailSent)
+        {
+            $this->response =
+            [
+                'message' => 'Invitation was sent successfully',
+                'status_code' => 201
+            ];
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show()
+    {
+        $roles = Role::get();
+
+        return view('dashboard.pages.create_user', compact('roles'));
+    }
+
+    /**
+     * Update user role
+     *
+     * @param  $username
+     * @param  $role_id
+     */
+    public function updateUser($username, $role_id)
+    {
+        $updateUserRole = User::where('username', $username)->update(['role_id' => $role_id]);
+
+        if ($updateUserRole)
+        {
+            return $this->deleteToken($username);
+        }
+    }
+
+    /**
+     * Delete user from invite table
+     *
+     * @param  $token
+     */
+    public function deleteToken($username)
+    {
+        $deleteUser = Invite::where('username', $username)->delete();
+
+        if ($deleteUser)
+        {
+            return redirect('/welcome/'.$username);
+        }
+    }
+
+    /**
+     * process the token from the email when clicked
+     *
+     * @param  $token
+     */
+    public function processInvite($token)
+    {
+        $checkToken = Invite::where('token', $token)->first();
+
+        if ($checkToken)
+        {
+            $this->response = $this->updateUser($checkToken->username, $checkToken->role_id);
+        }
+        else
+        {
+            $this->response = redirect('/invalid');
+        }
+
+        return $this->response;
+    }
+
+    /**
+     * Pass users and roles to edit_user view
+     *
+     * @param  int  $id
+     */
+    public function editView($id)
+    {
+        $users = User::where('id', $id)->first();
+        $roles = Role::get();
+
+        return view('dashboard.pages.edit_user', compact('users', 'roles'));
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     */
+    public function update(Request $request)
+    {
+        $updateUser = User::where('id', $request->user_id)->update(['role_id' => $request->user_role, 'username' => $request->username]);
+
+        if ($updateUser)
+        {
+            $this->response =
+            [
+                'message' => 'User details updated successfully',
+                'status_code' => 201
+            ];
+        }
+        else
```

Comment:
Fix this: the else should be on the same line with the closing brace of the if. Check out the PSR-2 guide [http://www.php-fig.org/psr/psr-2/](http://www.php-fig.org/psr/psr-2/)